sync-service/src/main/resources/application.yaml

```diff
@@ -0,0 +1,19 @@
+spring:
+  cloud:
+    loadbalancer:
+      ribbon:
+        enabled: false
+  cache:
+    type: simple
+
+feign:
+  client:
+    config:
+      default:
+        connectTimeout: 5000
+        readTimeout: 5000
+        loggerLevel: basic
+
+stayhome:
```

Comment:
I think this should use environment variables and not be defined in a static file.